requirements.txt

```
@@ -1,3 +1,4 @@
+hydra-genetics==0.0.4
```

```suggestion
hydra-genetics==0.0.5
```
version with get_sample function